ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/main.yml

```diff
@@ -12,6 +12,15 @@
   when:
     - ansible_architecture == "amd64"
 
+#########################################
+# Set ansible_processor_vcpus for s390x #
+#########################################
+- name: If ansible_architecture is s390x set ansible_processor_vcpus to ansible_processor_core
+  set_fact:
+    ansible_processor_vcpus: "{{ ansible_processor_cores }}"
+  when:
+    - ansible_architecture == "s390x"
+
```

Comment:
Agree with @Haroon-Khel on this - if we can add `ansible_processor_vpus is undefined` or whatever the valid syntax is to this `when` section that would be good ... In fact it could replace the `s390x` check :-)

Reply:
Yes, as this problem only occurs on RHEL (i think) and not every distribution on `s390x`

Reply:
Commit in :-)